tasks/lib/phantomas.js

```diff
@@ -506,7 +506,7 @@ Phantomas.prototype.getImages = function() {
 
   return _.sortBy( files, _.bind( function( file ) {
     return +file.match(
-      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d-(\d*).png$/
+      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d-\d\d-\d\d-(\d*).png$/
```

Comment:
@stefanjudis, please note that phantomas triggers `filmStrip` event on each screenshot save. May be a better approach then trying to match these files.